docs/core/compatibility/3.1.md

```diff
@@ -21,16 +19,15 @@ If you're migrating to version 3.1 of .NET Core or ASP.NET Core, the breaking ch
 
 ## MSBuild
 
-- [Design-time builds only return top-level package references](#design-time-builds-only-return-top-level-package-references)
-
 [!INCLUDE [design-time-builds-return-top-level-package-refs](../../../includes/core-changes/msbuild/3.1/design-time-builds-return-top-level-package-refs.md)]
 
 ***
 
-## Windows Forms
+## SDK
 
-- [Removed controls](#removed-controls)
-- [CellFormatting event not raised if tooltip is shown](#cellformatting-event-not-raised-if-tooltip-is-shown)
+[Tool manifests in root folder](sdk/7.0/manifest-search.md)
+
+## Windows Forms
```

Comment:
The SDK content looks good to me - but I see changes to ASP.NET Core, MSBuild, and Windows Forms that I'm not qualified to speak to.